ligolang/src/test/zinc_test.ml

```diff
@@ -60,6 +60,7 @@ type test =
 let expect_simple_compile_to ?(dialect = Self_ast_imperative.Syntax.PascaLIGO) ?(index = 0)
     ?(initial_stack = []) ?expect_failure ?expected_output_env ?expected_output
     contract_file (expected_zinc : Zinc_types.program) : test =
+ let open Ligo_compile.Helpers in  
```

Comment:
Is this supposed to be included?

Reply:
Thanks for catching it

Reply:
It's coming from a commit https://github.com/marigold-dev/deku/pull/353/commits/a6d7e747970986e3d40406b6e586f41a50cc3f61 while rebasing. Removing it from this PR